dash-renderer/src/components/core/DocumentTitle.react.js

```diff
@@ -29,8 +31,10 @@ class DocumentTitle extends Component {
 
 DocumentTitle.propTypes = {
     isLoading: PropTypes.bool.isRequired,
+    config: PropTypes.object,
```

Comment:
```suggestion
    config: PropTypes.shape({update_title: PropTypes.string}),
```

`PropTypes.object` will work, but since we know the piece we want from inside `config` it's nice to specify it. Note I'm suggesting `shape`, not `exact`, because `exact` would require us to list _all_ of config and here we only care about this one part.

Reply:
I see...this is much more explicit. Thanks for suggesting the change. I didn't know that one could make such explicit specificifications for PropTypes.